src/components/book.css

```diff
@@ -48,11 +61,194 @@
   letter-spacing: normal;
 }
 
+.links > .book {
+  width: 3.688rem;
+  height: 1rem;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.CATEGORIES {
+  width: 6.563rem;
+  height: 1rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.profile {
+  width: 30px;
+  height: 30px;
+}
+
 .add {
-  margin-top: 20px;
+  font-size: 1.1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  color: #888;
 }
 
 form {
   display: flex;
   gap: 15px;
+  margin-top: 10px;
+}
+
+.bookCard {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  padding: 12px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.interaction {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+button {
+  border: transparent;
+  background-color: transparent;
+  margin: 0;
+  font-weight: normal;
+  cursor: pointer;
+}
+
+.Comments {
+  width: 4.563rem;
+  height: 1.188rem;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book-category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  width: 5.813rem;
+  height: 1.188rem;
+  margin: 0.25rem 0.875rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.progress {
+  width: 60px;
+}
+
+.middle {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly add the vertical line after this middle container, also your font-size is smaller than the actual font size. Please increase it to match the given design. 
> figma: 
![image](https://github.com/touredri/bookstore/assets/91486791/9e867281-e47f-47c8-9e8e-0c84477ff433)
> yours:
![image](https://github.com/touredri/bookstore/assets/91486791/203aed5a-de21-4da1-8b76-5504b8da4874)
> Zeplin font size reference:
![image](https://github.com/touredri/bookstore/assets/91486791/29edb19d-b2af-4872-beea-667c40458822)